src/doc/unstable-book/src/compiler-flags/cf-protection.md

```diff
@@ -0,0 +1,40 @@
+# `cf-protection`
+
+This option enables control-flow enforcement technology (CET) on x86; a more detailed description of
+CET is available [here]. Similar to `clang`, this flag takes one of the following values:
+
+- `none` - Disable CET completely (this is the default).
+- `branch` - Enable indirect branch tracking (`IBT`).
+- `return` - Enable shadow stack (`SHSTK`).
+- `full` - Enable both `branch` and `return`.
+
+[here]: https://www.intel.com/content/www/us/en/develop/articles/technical-look-control-flow-enforcement-technology.html
+
+This flag only applies to the LLVM backend: it sets the `cf-protection-branch` and
+`cf-protection-branch` flags on LLVM modules. Note, however, that all compiled modules linked
```

Comment:
This looks like a copy paste error since branch is listed twice:
```suggestion
`cf-protection-return` flags on LLVM modules. Note, however, that all compiled modules linked
```

Reply:
Thanks!